From 3e6ce4279d9d1facaf1a6ab91ef6bdd544ad8be9 Mon Sep 17 00:00:00 2001
From: Seungha Yang <seungha@centricular.com>
Date: Thu, 15 Jun 2023 01:47:37 +0900
Subject: [PATCH 145/232] va: Fix Windows build

DRM_FORMAT_MOD_LINEAR and DRM_FORMAT_MOD_INVALID are not defined
on Windows

Part-of: <https://gitlab.freedesktop.org/gstreamer/gstreamer/-/merge_requests/4864>
---
 gst-libs/gst/va/gstvaallocator.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/gst-libs/gst/va/gstvaallocator.c b/gst-libs/gst/va/gstvaallocator.c
index 20992456a..e678192b5 100644
--- a/gst-libs/gst/va/gstvaallocator.c
+++ b/gst-libs/gst/va/gstvaallocator.c
@@ -41,6 +41,9 @@
 #include <sys/types.h>
 #include <unistd.h>
 #include <libdrm/drm_fourcc.h>
+#else
+#define DRM_FORMAT_MOD_LINEAR  0ULL
+#define DRM_FORMAT_MOD_INVALID 0xffffffffffffff
 #endif
 
 #include "gstvasurfacecopy.h"
-- 
2.40.1

